inst/NEWS.md

```diff
@@ -1,4 +1,11 @@
 
+* ORCID support (@maelle, #70):
+    * Expose an orcid argument in several functions to add and search authors, 
+    * add an add_orcid function, 
+    * add_me now can use the `ORCID_ID` environment variable.
+
+* The comment field can now be a named character vector. (@maelle, #69; @gvegayon, #65)
+
```

Comment:
Can you pls rewrite this more in the style of http://style.tidyverse.org/news.html#before-release

New `add_orcid()` method and `desc_add_orcid()` functions...

Reply:
I tried :-)